chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
you mentioned "in this case submit button is out of form container"
I asked: "could you let me know how is the form submitted if the submit is out of the container?"
I'd genuinely like to know.